### PR TITLE
style(formatting): removing platform from introduction section

### PR DESCRIPTION
**Pull request by christinaausley**
Partially closes https://github.com/camunda/developer-experience/issues/117.

### CODE REVIEW

File: content/installation/camunda-bpm-run.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ menu:
     name: "Remote Engine Distribution"
     identifier: "installation-camunda-bpm-run"
     parent: "installation-guide"
-    pre: "Install Camunda Platform Run, an easy to configure remote engine distribution of the Camunda Platform. No Java knowledge necessary."
+    pre: "Install Camunda Platform Run, an easy to configure remote engine distribution of Camunda. No Java knowledge necessary."
 ---
 
+<!--- Do we want to remove Platform from Run as well? --->
```
Comment (tasso94) [inline content/installation/camunda-bpm-run.md:14]: I would tend to say yes. 🙂

File: content/installation/full/tomcat/manual.md [hunk 2]
```diff
@@ -83,20 +83,20 @@ For more information on the creation of JDBC datasources have a look at the docu
...
+## Add Camunda Services
 
 Copy the following blocks from `${TOMCAT_DISTRIBUTION}/server/apache-tomcat-${TOMCAT_VERSION}/conf/server.xml`
   into `${TOMCAT_HOME}/conf/server.xml`:
 
 ```xml
      <Resource name="global/camunda-bpm-platform/process-engine/ProcessEngineService!org.camunda.bpm.ProcessEngineService" auth="Container"
               type="org.camunda.bpm.ProcessEngineService"
-              description="Camunda Platform Process Engine Service"
+              description="Camunda Process Engine Service"
```
Comment (tasso94) [inline content/installation/full/tomcat/manual.md:94]:
```suggestion
              description="Camunda Platform Process Engine Service"
```
This is hard-coded into our Tomcat distribution. Can we leave it as it is?

File: content/installation/full/tomcat/manual.md [hunk 3]
```diff
@@ -83,20 +83,20 @@ For more information on the creation of JDBC datasources have a look at the docu
...
               factory="org.camunda.bpm.container.impl.jndi.ProcessEngineServiceObjectFactory" />
 
     <Resource name="global/camunda-bpm-platform/process-engine/ProcessApplicationService!org.camunda.bpm.ProcessApplicationService" auth="Container"
               type="org.camunda.bpm.ProcessApplicationService"
-              description="Camunda Platform Process Application Service"
+              description="Camunda Process Application Service"
```
Comment (tasso94) [inline content/installation/full/tomcat/manual.md:99]:
```suggestion
              description="Camunda Platform Process Application Service"
```
This is hard-coded into our Tomcat distribution. Can we leave it as it is?

**Review by tasso94 — CHANGES_REQUESTED** [review]
Looks good to me 👍 
Just some minor hints.

**Comment by akeller** [comment]
I promise I am desperately trying not to overcomplicate this 😰 

When reviewing this PR, I am wondering about SEO and continuing to differentiate C7 from C8. Looking at:

> Install Camunda

Has me wondering if we need to add a "7" here. It's included in other areas of these docs. Certain terms and concepts should be pretty good indicators between what is C7 and C8 content, we but have so many new and prospective customers that I think we need to be really explicit in our meta data (titles and pre). 

@christinaausley @tasso94 WDYT? Is this too much or too detailed? 

Outside of this aspect, I agree with @tasso94's feedback. Let's leave any code snippets or config as-is.

**Comment by christinaausley** [comment]
> I promise I am desperately trying not to overcomplicate this 😰
> 
> When reviewing this PR, I am wondering about SEO and continuing to differentiate C7 from C8. Looking at:
> 
> > Install Camunda
> 
> Has me wondering if we need to add a "7" here. It's included in other areas of these docs. Certain terms and concepts should be pretty good indicators between what is C7 and C8 content, we but have so many new and prospective customers that I think we need to be really explicit in our meta data (titles and pre).
> 
> @christinaausley @tasso94 WDYT? Is this too much or too detailed?
> 
> Outside of this aspect, I agree with @tasso94's feedback. Let's leave any code snippets or config as-is.

I'm comfortable adding the "7" for clarity -- will adjust!

File: content/installation/camunda-bpm-run.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
 ---
 
-title: "Install Camunda Platform Run"
+title: "Install Camunda Run"
```
Comment (akeller) [inline content/installation/camunda-bpm-run.md:3]: I mentioned this on the other PR, but we probably want to add a 7 to the title here so we can leave room for a C8 run.

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
 ---
 
-title: 'Camunda Platform License Book'
+title: 'Camunda License Book'
```
Comment (akeller) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:3]:
```suggestion
title: 'Camunda 7 License Book'
```

File: content/installation/karaf-osgi.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 
-title: "Run Camunda Platform using Karaf / OSGi"
+title: "Run Camunda using Karaf / OSGi"
```
Comment (akeller) [inline content/installation/karaf-osgi.md:3]:
```suggestion
title: "Run Camunda 7 using Karaf / OSGi"
```

File: content/installation/full/wls/manual.md [hunk 7]
```diff
@@ -138,18 +138,18 @@ First the `camunda-oracle-weblogic-ear-$PLATFORM_VERSION.ear` RAR file must be i
...
 
 **Please note:** If you change the name of the enterprise application deployment, the JNDI names change as well.
 {{< /note >}}
 {{< img src="../img/ear-name.png" title="EAR Name" >}}
-7. Continue to the **"Next"** page.
-8. Select **"Yes, take me to the deployment's configuration screen"** and click the **"Finish"** button to complete the installation.
-9. (optional) [Configure location of the bpm-platform.xml file]({{< ref "/reference/deployment-descriptors/descriptors/bpm-platform-xml.md#configure-location-of-the-bpm-platform-xml-file" >}}).
+1. Continue to the **"Next"** page.
+2. Select **"Yes, take me to the deployment's configuration screen"** and click the **"Finish"** button to complete the installation.
+3. (optional) [Configure location of the bpm-platform.xml file]({{< ref "/reference/deployment-descriptors/descriptors/bpm-platform-xml.md#configure-location-of-the-bpm-platform-xml-file" >}}).
```
Comment (akeller) [inline content/installation/full/wls/manual.md:152]: Looks like this was an editor linting add that you don't want to happen
```suggestion
7. Continue to the **"Next"** page.
8. Select **"Yes, take me to the deployment's configuration screen"** and click the **"Finish"** button to complete the installation.
9. (optional) [Configure location of the bpm-platform.xml file]({{< ref "/reference/deployment-descriptors/descriptors/bpm-platform-xml.md#configure-location-of-the-bpm-platform-xml-file" >}}).
```

File: content/installation/full/was/manual-liberty.md [hunk 8]
```diff
@@ -247,7 +247,7 @@ You need to perform the following steps to install the EAR archive on WebSphere
 3. Define an `enterpriseApplication` element in the `server.xml`.
 4. Reference the Camunda shared library in the `enterpriseApplication` inside the `server.xml`.
 * The Camunda shared library is referenced by adding a `classloader` element.
-5. (optional) [Configure location of the `bpm-platform.xml` file][bpm-platform-xml-config].
+1. (optional) [Configure location of the `bpm-platform.xml` file][bpm-platform-xml-config].
```
Comment (akeller) [inline content/installation/full/was/manual-liberty.md:250]:
```suggestion
5. (optional) [Configure location of the `bpm-platform.xml` file][bpm-platform-xml-config].
```

File: content/installation/full/was/manual-liberty.md [hunk 9]
```diff
@@ -23,25 +23,25 @@ you will need the following:
...
-To fully install Camunda Platform 7 on WebSphere Liberty, you need to configure the following components:
+To fully install Camunda 7 on WebSphere Liberty, you need to configure the following components:
 
 1. Camunda EAR and shared libraries
-1. Datasource
-1. Work Manager
-1. Optional components: Camunda web apps, REST API, etc.
+2. Datasource
+3. Work Manager
+4. Optional components: Camunda web apps, REST API, etc.
```
Comment (akeller) [inline content/installation/full/was/manual-liberty.md:44]: ✅ Good catch

File: content/installation/camunda-bpm-run.md (hunk 4 above)
Comment (akeller) [reply content/installation/camunda-bpm-run.md:3]:
```suggestion
title: "Install Camunda 7 Run"
```

**Review by akeller — CHANGES_REQUESTED** [review]
Double check the lists to make sure they didn't get caught and incorrectly reset. I added a 7 as a suggestion in a few places. Beyond that, this looks good.

**Review by tasso94 — APPROVED** [review]
Looks good to me 🙂 👍

**Review by akeller — APPROVED** [review]

**Pull request merged**